src/action_types/actionTypes.ts

```diff
@@ -1,5 +1,12 @@
 export const ASYNC_ACTION_START = 'common/ASYNC_ACTION_START';
 export const ASYNC_ACTION_END = 'common/ASYNC_ACTION_END';
+export const CONFIRM_ACTION = 'common/CONFIRM_ACTION';
```

Comment:
For new structure we do separate action types into different files

Please see new template structure for reference